### DISCUSSION TITLE
Feature Request: Wish Rank/Priority

### DISCUSSION THREAD

**Discussion by tpansino** [cmintey/wishlist#117] [Ideas]
Inspired by [Wishthis](https://github.com/wishthis/wishthis)...

```
As a user
I want to be able to rank or prioritize items in my wishlist
So that I can convey to my friends and family which items I am most interested in.
```

Wishthis does this by having 3 optional preset Priorities:
- "Would love it"
- "Nice to have"
- "Unsure about it"

Items marked with one of these priorities are by default sorted higher in when displaying the list, and unprioritized items show at the bottom. For example:

![Screenshot 2024-07-20 103651](https://github.com/user-attachments/assets/b4579cb2-915c-4b47-b788-f32c8918b288)

Using the tool, I have found the granular categories like this to be a more pleasant UX than say, a numeric rank system, where every item must have a rank value, but that would be of course a valid implementation here as well.

Thanks very much for all of your efforts on this project, looks great. Let me know if I can clarify anything about the idea.

**Comment by cmintey** [comment]
Thanks for the feature request! I have wanted to be able to re-order items in a list for a while, but haven't come up with a great UX design for it. I quite like the idea behind a priority tag. I may implement something similar to that

**Comment by cmintey** [comment]
Hey @tpansino. I just released an update which allows you to re-order the items on your wishlist!

**Reply by tpansino** [reply]
Looks good to me! Thanks @cmintey!
